### PR TITLE
addressVo updated with area/state/city/country

### PR DESCRIPTION
**Pull request by jitendrajaria**
addressVo updated with hql query

### CODE REVIEW

File: src/main/java/creativei/helper/ResponseHelper.java [hunk 1]
```diff
@@ -187,5 +195,10 @@ public static FollowUpVo getCreateFollowUpData(FollowUp followUp, FollowUpVo fol
         return followUpVo;
     }
 
+    public static LocalityVo getCreateLocalityVo(Inquiry inquiry, InquiryVo inquiryVo) {
+        Locality locality = inquiry.getInquiryAddress().getLocation();
+        return null;
```
Comment (aayushKhandpur) [inline src/main/java/creativei/helper/ResponseHelper.java]: Why return null from here?

File: src/main/java/creativei/manager/impl/LocalityMangerImpl.java [hunk 2]
```diff
@@ -29,7 +30,7 @@ public ResponseObject getPincodes(String pincode) {
 
     @Override
     public ResponseObject getLocalityDataByPincode(String pincode) {
-        CityVo city=new CityVo();
+        CityVo city = new CityVo();
```
Comment (aayushKhandpur) [inline src/main/java/creativei/manager/impl/LocalityMangerImpl.java]: Move city specific initialization code to CityVo and add constructor there. This way you can remove all the setters being called from this method.
CityVo(City city){

}

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review comments

File: src/main/java/creativei/helper/ResponseHelper.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/helper/ResponseHelper.java]: sir this is dummy method use for testing i forget to remove it.Remove in commit 4ab6df7

File: src/main/java/creativei/manager/impl/LocalityMangerImpl.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/manager/impl/LocalityMangerImpl.java]: ok. Issue resolved in commit 4ab6df7

File: src/main/java/creativei/entity/Locality.java [hunk 3]
```diff
@@ -1,13 +1,14 @@
 package creativei.entity;
 
+import creativei.manager.impl.LocalityMangerImpl;
```
Comment (aayushKhandpur) [inline src/main/java/creativei/entity/Locality.java:3]: Remove import

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**